src/backend/access/spgist/spgscan.c

```diff
@@ -585,7 +591,7 @@ spggetbitmap(PG_FUNCTION_ARGS)
 
 	spgWalk(scan->indexRelation, so, true, storeBitmap);
 
-	PG_RETURN_INT64(so->ntids);
+	PG_RETURN_POINTER(tbm);
```

Comment:
Maybe explain a bit more on this?

Reply:
It's to keep the same as xxgetbitmap (btgetbitmap/gistgetbitmap/gingetbitmap)